locale/en/about/releases.md

```diff
@@ -20,6 +20,6 @@ schedule-footer: Dates are subject to change.
 # Releases
 
 Major Node.js versions enter _Current_ release status for six months, which gives library authors time to add support for them.
-After six months, odd-numbered releases (9, 11, etc.) become unsupported, and even-numbered releases (10, 12, etc.) move to _Active LTS_ status and are ready for general use.
+After six months, odd-numbered releases (13, 15, etc.) become unsupported, and even-numbered releases (14, 16, etc.) move to _Active LTS_ status and are ready for general use.
```

Comment:
What about adding "e.g:" before 13,15 and 14,16? Because we don't want to bump the versions again and again with the upgrade of Node.js versions.

After six months, odd-numbered releases (e.g: 13, 15, etc.)

Reply:
Sorry, I'm no longer able to edit it. But I can come back two years later and manually change it again)

Reply:
@web2033: No problem, If you agree, I'll change it for you if possible, waiting for your resp:)

Reply:
Sure.

Reply:
Well……It seems I cannot edit the file....Can you allow me to edit it by clicking "Allow to Edit.." at the right-down corner?
![image](https://user-images.githubusercontent.com/52018749/133966349-8dd6e38f-52dc-498d-80ca-77d3857e99b7.png)

Reply:
I don't see any checkboxes available to me...